### DISCUSSION TITLE
Custom RGB Console Color?

### DISCUSSION THREAD

**Discussion by clveranis** [spectreconsole/spectre.console#692] [Q&A] [answered]
Is it possible to change the entire background color of the console to a custom RGB color?

I've tried something like 

`AnsiConsole.Background = new Color(34,38,50)`

But that doesn't work unfortunately. I know that I can set it using `ConsoleColor` but the options are limited so I wasn't sure if Spectre Console was able to provide more options.

Thanks in advance!

**Comment by patriksvensson** [comment, marked as answer]
@clveranis It's up to the console you're using, how colors are displayed and what bitness it supports.

Sadly, nothing Spectre.Console (or any other library) can do about.

**Reply by clveranis** [reply]
Ah, okay, I gotchya. Well thank you for the quick reply!
